src/tests/unit/pizzas-dao.test.js

```diff
@@ -189,4 +204,114 @@ describe('test pizzas DAO', () => {
       });
     });
   });
+  context('getPizzas', () => {
```

Comment:
This looks like one really long block of code. Use whitespace to break it up and make it more readable.